### DISCUSSION TITLE
using js variables in css

### DISCUSSION THREAD

**Discussion by tryggs** [sveltejs/kit#5560] [Q&A] [answered]
Hello there! I know that to use css from an outside URL I use this in svelte:
`<style>
@import url("https://www.example.com/style.css");
</style>
`

but what if that URL Is saved in a js variable called _testurl_? 
i tried doing this:
`@import url({testurl});
`
but it didn't work...
any ideas?

**Comment by mustofa-id** [comment, marked as answer]
JavaScript variable not accessible to `<style ...>` scope. If you need to change css url dynamically, you can use `<link ...>` in `<svelte:head ...>` instead. Here is example: https://svelte.dev/repl/0c3e53d75e364e4a8903d1033c17f6fd?version=3.49.0

**Reply by mustofa-id** [reply]
Also a nice feature of Svelte might be related: [css var](https://svelte.dev/docs#:~:text=Svelte%27s%20CSS%20Variables)

**Reply by tryggs** [reply]
thank you!

**Reply by mustofa-id** [reply]
Glad to help
